src/py42/sdk/queries/fileevents/filters/cloud_filter.py

```diff
@@ -3,23 +3,49 @@
 
 
 class Actor(FileEventFilterStringField):
+    """Class that filters events by the cloud service username of the event originator
+    (applies to cloud data source events only).
+    """
+
     _term = u"actor"
 
 
 class DirectoryID(FileEventFilterStringField):
+    """Class that filters events by unique identifier of the cloud drive or folder where the event
+     occurred (applies to cloud data source events only).
+    """
+
     _term = u"directoryId"
 
 
 class Shared(QueryFilterBooleanField):
+    """Class that filters events by the shared status of the file at the time the event occurred
+    (applies to cloud data source events only).
+    """
+
     _term = u"shared"
 
 
 class SharedWith(FileEventFilterStringField):
+    """Class that filters events by the users files were explicitly shared with at the time of the
+    event (applies to cloud data source events only).
+    """
+
     _term = u"sharedWith"
 
 
 class SharingTypeAdded(FileEventFilterStringField):
+    """Class that filters results to include events where a file's sharing permissions were
+    increased to increase exposure (applies to cloud data source events only).
+
+    Available options are:
+        - :class:`SharingTypeAdded.SHARED_VIA_LINK`
```

Comment:
Is this supposed be :attr: instead of :class: ?

Reply:
yep, I think I wrote that before I realized :attr: was an option.